src/System.Runtime/tests/System/String.cs

```diff
@@ -19,7 +20,7 @@ public static void TestLastIndexOfWithEmptyString()
     {
         string s = "Dill Guv Dill Guv Dill";
         int lastIndex = s.LastIndexOf("", StringComparison.OrdinalIgnoreCase);
-        Assert.True(lastIndex == s.Length - 1);
+        Assert.Equal(lastIndex, s.Length - 1);
```

Comment:
Swapped the arguments + updated some of the existing ones that were in the wrong order.